README.md

```diff
@@ -1,69 +1,137 @@
+mission: ORCESTRA
+platform: HALO
+flight_id: HALO-20240816a
+takeoff: 2024-08-16 11:06:34
+landing: 2024-08-16 21:59:33
+segments:
+- kinds:
+  - straight_leg
+  name: ferry ascent
+  segment_id: HALO-20240816a_kd78
+  start: 2024-08-16 11:39:05
+  end: 2024-08-16 12:06:58
+  irregularities: [strange spike in roll angle at 12:01:37]
+  comments: []
+- kinds:
+  - straight_leg
+  - ec_track
+  name: ec track southwards
+  segment_id: HALO-20240816a_0k2e
+  start: 2024-08-16 12:06:58
+  end: 2024-08-16 12:51:55
+  irregularities: []
+  comments: []
+```
+
+
+**Note:** Time ranges are defined as semi-open intervals, i.e. while the start time is inside the segment, the end time is not. This allows for an unambiguous definition of exactly consecutive segments. In the case of underpass and overpass segments, the start and end times are identical.
+
+## Definition of kinds
+Kinds may or may not apply/be adopted for all platforms. For version 1 of the segmentation of HALO flights, one or more of the following kinds are assigned to each segment:
+
+### circle:
+- can be identified by a constant change of aircraft heading, lasting anywhere around 20 to 60 min (depending on circle radius), as well as a roughly constant roll angle of 1.5-3 degrees (positive or negative, depending on turning clockwise or counter-clockwise) for the standard circle of about 200km diameter. The smaller ATR circle has a larger roll angle of 3-5 degree depending on and varying with the wind speed and direction at the respective altitude.
+- in most cases associated with the launch of dropsondes, typically 12 per circle at every 30&deg; heading. In cases where the first/last dropsonde of the circle was launched before/after the roll angle and change in heading match the circle characteristics described above, the start/end time of the circle segment is taken to be the `launch_time` of the respective dropsonde minus/plus 1 sec. 
+- Circle dropsondes were sometimes droped before/after the roll angle reached its circle value in which case the launch time of the sonde  marks the beginning/end of the circle period (minus/plus a few seconds, so that the sonde is included in the circle segment). 
+- Some circles were highly irregular due to detours around convective towers. In these cases, an irregularity note need to be added.
 
-### Common vocabulary - broad sampling strategy
-Following are names of segments assembled from a range of platforms describing broad sampling strategies. Many will contain subsets
-e.g. `cloud` segments will likely contain `subcloud layer` and `cloud layer` segments.
-Data providers are encouraged to use these names, especially those in the first, where applicable.
-Feel free to add to the list (e.g. with a Github pull request) if needed.
-* ground
-* transit
-* circle
-* cloud
-* calibration
-* profile
-* axbt
-* rectangle
-* sawtooth
-* racetrack
+### ATR_coordination:
+- circle that was also flown by the ATR aircraft on the same day
```

Comment:
The ATR did not fly a circle, but rather performed measurements within the same airmass
```suggestion
### atr_coordination:
- HALO flew several 72km radius circles in coordination with parallel (or slightly delayed) ATR measurements that aimed at sampling the same airmass. 
```